src/api/createVoiceVideoCalling.ts

```diff
@@ -340,11 +340,15 @@ export class VoiceVideoCallingProxy {
                 SecondaryChannelEventType: SecondaryChannelEvents.End
             }
 
+            this.scenarioMarker?.startScenario(TelemetryEvent.onCallDisconnected);
```

Comment:
We need to pass `callId`. You can refer to [this](https://github.com/microsoft/omnichannel-chat-sdk/blob/f9059fb6eec2623470efb30fce52e630a4a7d9dc/src/api/createVoiceVideoCalling.ts#L135)

```
        this.scenarioMarker?.startScenario(params.withVideo? TelemetryEvent.AcceptVideoCall: TelemetryEvent.AcceptVoiceCall, {
            CallId: callId || ''
        });
```